README.md

```diff
@@ -14,31 +13,43 @@ Este repositório contém o código do nosso website [raparigasdocodigo.pt](http
-| `npm run buil:deploy-ftp` | [Verificar a secção de implementação](/docs/DEPLOYMENT.md#deployment-process). |
+| Comando                   | Descrição                                                                                                                                                                                                              |
+| ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `npm install`             | Instala as dependências necessárias para executar a aplicação.                                                                                                                                                         |
+| `npm start`               | Compila a aplicação em modo de desenvolvimento. Abre [http://localhost:3000](http://localhost:3000) para visualizares no teu browser.                                                                                  |
+| `npm run build`           | Compila a app na pasta `build`. Isto integra corretamente a biblioteca React no modo de produção e otimiza a compilação para o melhor desempenho. A compilação é minificada e os nomes dos arquivos incluem os hashes. |
+| `npm run buil:deploy-ftp` | [Verificar a secção de implementação](/docs/DEPLOYMENT.md#deployment-process).                                                                                                                                         |
+| `npm run lint`            | Corre o linter.                                                                                                                                                                                                        |
+| `npm run lint:fix`        | Corre o linter e corrige os erros possíveis de correção (Em caso de dúvidas, verificar [neste site](https://eslint.org/docs/latest/rules/)).                                                                           |
+| `npm run format`          | Corre o formatador em todos os arquivos.                                                                                                                                                                               |
+| `npm run precommit`       | Corre o linter, corrige os erros possíveis e formata o código.                                                                                                                                                         |
+| `npm run configure-husky` | Configura o husky, se não estiver configurado no projeto.                                                                                                                                                              |
```

Comment:
@juliamendesc só para ver se percebo, como nova colaboradora, para isto funcionar logo, eu tenho de correr este comando, ou posso começar a fazer alterações no projeto, e quando fizer commit o lint é fixo. Preciso de acionar alguma coisa?

Reply:
Pode trabalhar normalmente e o script vai correr automaticamente quando for fazer commit. Se tiver erros de lint que podem ser corrigidos, ele vai corrigir automaticamente. Se não tiver correção automática, ele vai abortar o commit e o developer vai ter que resolver o problema para conseguir fazer o commit.

Reply:
Oi @juliamendesc Obrigada pela contribuicao aqui! Eu nao tou a conseguir correr o projeto localmente a partir do branch `dev`, não sei se me está a faltar alguma coisa. Criei issue aqui pra ver o que falta https://github.com/As-Raparigas-do-Codigo/ardc-website/issues/143 não tem qualquer expectativa de ires lá, só tou a avisar porque mencionei este PR. 

Criei nove issue, também porque assim podemos perceber se há alguma coisa a melhorar no README, caso me tenha faltado algum passo.